models/demos/tg/llama3_70b/tt/llama_attention_galaxy.py

```diff
@@ -524,16 +545,16 @@ def prefill_attn_qkv(
             dtype=ttnn.bfloat16,
             memory_config=ttnn.DRAM_MEMORY_CONFIG,
             compute_kernel_config=self.COMPUTE_KERNEL_QKV,
+            program_config=self.FUSED_QKV_MM_PROGCFG,
         )
 
         fused_query_key_value = tt_all_reduce(
             fused_query_key_value, self.mesh_device, cluster_axis=1, num_links=2, memory_config=ttnn.DRAM_MEMORY_CONFIG
         )
+        # fused_query_key_value.deallocate(True)
 
         fused_query_key_value = ttnn.reshape(fused_query_key_value, (1, 1, seq_len, -1))
 
-        xs.deallocate(True)
```

Comment:
We figured out that deallocation is not needed since even for 80 layers memory is self-deallocated soon enough so no need to call it from our side. Also, we suspected that deallocation might cause a hang since it happened earlier in falcon to me so we decided to try out without it and since there were no issues we left deallocation to garbage collector.